MailCore/Cache/MailboxInfosManager.swift

```diff
@@ -123,7 +124,7 @@ public final class MailboxInfosManager {
     public func removeMailboxesFor(userId: Int) {
         let realm = getRealm()
         let userMailboxes = realm.objects(Mailbox.self).where { $0.userId == userId }
-        try? realm.uncheckedSafeWrite {
```

Comment:
Only the `uncheckedSafeWrite` was used in mail and this is 100% the `safeWrite` present in `core-db`